server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -221,20 +221,20 @@ class ChallengeFlow extends BaseStep {
         }
 
         this.challenge.claim = this.challenge.getClaim();
-        this.game.promptWithMenu(this.challenge.winner, this, {
+        this.game.promptWithMenu(this.challenge.loser, this, {
             activePrompt: {
-                menuTitle: 'Perform before claim actions',
+                menuTitle: 'Claim will be applied',
                 buttons: [
-                    { text: 'Apply Claim', method: 'applyClaim' },
-                    { text: 'Continue', method: 'cancelClaim' }
+                    { text: 'Continue', method: 'applyClaim' },
+                    { text: 'Cancel', method: 'cancelClaim' }
```

Comment:
I understand the information hiding arguments for prompting the losing player instead of the winner, but I'm not sure allowing them to cancel claim application is a good idea. Seems like a good way to spawn even more arguments (trolls just refusing to allow claim to be applied). It also could cause confusion - if you want to play Vengeance for Elia, for instance, you should click 'Continue', but one might think you should click 'Cancel' because you're replacing claim (in which case Vengeance won't fire because claim wasn't applied).

I'd propose that there should be only one button - one that applies claim. And perhaps as a future PR, make this prompt optional for decks that don't care about information hiding during claim apply. I defer to @cryogen though.